README.md

```diff
@@ -21,6 +21,16 @@ Thanks my mentor [@jerome1210](https://github.com/Jerome1210)
 ___
 
 ### Learning Note
+### :pushpin: 21.Mar.2023
+**mongoDB**
+- [lookup: left join](https://www.youtube.com/watch?v=kLlpIuyL8Ms)
+- send Data API request with mongosh
+- connect to mongoDB with Node.js driver
+
+**Project: MoneyList**
+- create new repo
+- connect local to GitHub
+
```

Comment:
I think you should record the git problem you met and how you solve to make it a reference for the future.

Reply:
update problem and solution in readme